Automattic/wp-calypso
# Use a static image for GoDaddy GoCentral logo.

Before this change, the logo was a PNG image encoded in Base64, embedded in SVG, and transpiled to JS.

Besides this being very inefficient at runtime for the browser, it ended up taking a large portion of the importer chunk, increasing loading times and affecting mobile users with unnecessary CPU usage.

@Automattic/team-calypso : can you please confirm this is the correct way of including a new static resource?
I'm guessing cache-busting can manually be added to the requested URL in JSX if the resource is ever updated?
Also, is hardcoding the full path like I did the right way of doing things, or should I be using some routing function to calculate the path?

#### Changes proposed in this Pull Request

* Use a static image for GoDaddy GoCentral logo instead of embedding it in JSX
* Optimise the image, bringing it down to ~25KB from ~70KB

#### Testing instructions

* Go to `/settings/import`
* Ensure the GoDaddy GoCentral logo loads correctly and looks correct.
